lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -98,12 +99,13 @@ def get_tick_iterators(self, axes):
 
     class _Base:
         """Base class for axis helper."""
-        def __init__(self):
-            self.delta1, self.delta2 = 0.00001, 0.00001
 
         def update_lim(self, axes):
             pass
 
+        delta1 = _api.deprecated("3.6")(property(lambda self: 0.00001))
```

Comment:
If I understand correctly, this means that users get a deprecation when accessing the values, but trying to set my own value will fail hard immediately?

Reply:
Fair point, fixed (the setter is now a noop).